### PR TITLE
Lecture 13 and data

### PR DESCRIPTION
**Pull request by mbonsma**
Is this file too large to go in our repo? Should I try to remove some columns and make it smaller?

### CODE REVIEW

**Comment by James-S-Santangelo** [comment]
3.3 Mb doesn't seem to large to me. For my lecture using NEON data I subsetted the data to only use a portion of it in class. Alternatively, you can download the data in your script using the data utility R package provided by NEON. There is an example [here](https://www.neonscience.org/neonDataStackR).

**Comment by mbonsma** [comment]
It's already heavily subsetted, so if I use the NEON utilities I'll be getting waaay more than I want. But if you think it's alright I'll just leave it as is.

**Comment by QuLogic** [comment]
GitHub hard limit is 100MB, with a warning over 50, I think. 3.3MB is not that big...

File: lec13-time-series.Rmd [hunk 1]
```diff
@@ -0,0 +1,940 @@
...
+
+```{r, eval=FALSE}
+?flowField # the nice thing about this packages is that it takes functions in the same format as `ode`
+```
+
+Let's plot a vector field and some trajectories for the predator-prey model.
+
+```{r}
+# plot vector field: on a grid of points, plot an arrow in the direction of dx/dt and dy/dt
+pp_flowField <- flowField(predator_prey, x.lim = c(0, 10), y.lim = c(0, 10),
```
Comment (aays) [inline lec13-time-series.Rmd]: `xlim = c(0, 10), ylim = c(0, 10)` - an update to `phaseR` changed the names of these arguments, which is what's breaking Travis (and also breaking on my end)

File: lec13-time-series.Rmd [hunk 2]
```diff
@@ -0,0 +1,940 @@
...
+                          points = 15, # this is the density of grid points on which to plot arrows
+                          system = 'two.dim', # 'two.dim' is default
+                          add = FALSE)
+
+# add trajectories
+pp_trajectory <- trajectory(predator_prey, 
+                            # y0 is a matrix where each row is pairs of (X, Y) 
+                            y0 = matrix(c(1, 1, 8, 5, 2, 0, 0, 8), ncol = 2,
+                                   byrow = TRUE), 
+                            t.end = 20, # how far in time to calculate the trajectories
```
Comment (aays) [inline lec13-time-series.Rmd]: `tlim = c(0, 20)` - this is another syntax change in `phaseR`, and was breaking a lecture from last year (`lec06` iirc) that I tested and fixed back in August

File: lec13-time-series.Rmd [hunk 3]
```diff
@@ -0,0 +1,940 @@
...
+
+Use `phaseR` and the function `flowField` to make a phase portrait for the mass-spring system 
+of differential equations. You will need to use the function you wrote in the previous challenge.
+Is there a fixed point, and if so, where do you think it is?
+
+```{r, echo=FALSE, eval=FALSE}
+# Challenge solution
+
+# plot vector field: on a grid of points, plot an arrow in the direction of dx/dt and dv/dt
+mass_spring_flowField <- flowField(mass_spring, x.lim = c(-3,3), y.lim = c(-2,2),
```
Comment (aays) [inline lec13-time-series.Rmd]: `xlim = c(-3,3), ylim = c(-2,2)`

File: lec13-time-series.Rmd [hunk 4]
```diff
@@ -0,0 +1,940 @@
...
+                          points = 21, 
+                          system = 'two.dim', # 'two.dim' is default
+                          add = FALSE)
+
+# add trajectories - not part of challenge, just for fun.
+mass_spring_trajectory <- trajectory(mass_spring, 
+                            # y0 is a matrix where each row is pairs of (X, Y) 
+                            y0 = matrix(c(1, 1, 0, 1, 2, -1.5), ncol = 2,
+                                 byrow = TRUE), 
+                            t.end = 20, # how far in time to calculate the trajectories
```
Comment (aays) [inline lec13-time-series.Rmd]: `tlim = c(0, 20)`

File: lec13-time-series.Rmd (hunk 2 above)
Comment (aays) [reply lec13-time-series.Rmd]: Fixed for last year's lectures in PR #203 - I realize now I should have done this in a separate PR...

**Pull request merged**